gitpod/gitpod.Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Doing a local shallow clone - keeps the container secure
+# and much slimmer than using COPY directly or making a
+# remote clone
+ARG BASE_CONTAINER="pythonpandas/pandas-dev:latest"
+FROM gitpod/workspace-base:latest as clone
+
+# the clone should be deep enough for versioneer to work
+RUN git clone https://github.com/pandas-dev/pandas --depth 12 /tmp/pandas
```

Comment:
Is depth of 12 then deep enough? Or will this still automatically fetch tags? (otherwise you can also explicitly fetch tags in a separate command)

Reply:
I thought it should be when we last chatted about this?!